File: backend/internal/database/models/founders.go
package models

import "time"

type Founder struct {
	ID        uint64    `json:"id" gorm:"primaryKey"`
	UserID    uint64    `json:"user_id"`
	StartupID uint64    `json:"startup_id"`
	CreatedAt time.Time `json:"created_at"`
}

File: web/src/lib/api/contracts/founders.ts
export type FounderDTO = {
  id: number;
  user_id: number;
  startup_id: number;
  created_at: string;
};

export type Founder = {
  id: number;
  userId: number;
  startupId: number;
  createdAt: Date;
};

export const mapFounder = (dto: FounderDTO): Founder => ({
  id: dto.id,
  userId: dto.user_id,
  startupId: dto.startup_id,
  createdAt: new Date(dto.created_at),
});
